pimd/pim_pim.c

```diff
@@ -717,19 +718,19 @@ int pim_msg_send(int fd, pim_addr src, pim_addr dst, uint8_t *pim_msg,
 		pim_pkt_dump(__func__, pim_msg, pim_msg_size);
 	}
 
-	pim_msg_send_frame(fd, (char *)buffer, sendlen, (struct sockaddr *)&to,
-			   tolen, ifp->name);
-	return 0;
+	ret = pim_msg_send_frame(fd, (char *)buffer, sendlen,
```

Comment:
Do we need a ret variable here? We can directly return the function. Otherwise coverity or some static tool will complain about not using ret variable.
return pim_msg_send_frame(fd, (char *)buffer, sendlen, (struct sockaddr *)&to, tolen, ifp->name);